### PR TITLE
Optimize tests and import

### PR DESCRIPTION
**Pull request by birdstorm**
optimize some import in TiSpark and fix incorrect test output.

### CODE REVIEW

File: core/src/main/scala/com/pingcap/tispark/TiDBRelation.scala [hunk 1]
```diff
@@ -93,9 +93,9 @@ case class TiDBRelation(session: TiSession,
       NamedExpression.newExprId
     )
 
-    val sortAgg = TiUtil
+    val agg = TiUtil
```
Comment (birdstorm) [inline core/src/main/scala/com/pingcap/tispark/TiDBRelation.scala]: FYI, this is not a sort aggregation.

File: core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala [hunk 2]
```diff
@@ -363,14 +361,33 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
...
       try {
         r1 = queryViaTiSpark(qSpark)
+        sparkPlan = getSparkPlan(qSpark)
       } catch {
         case e: Throwable =>
-          logger.error(s"TiSpark failed when executing: $qSpark")
-          fail(e)
+          try {
+            logger.error(s"TiSpark failed when executing: $qSpark")
+            logger.warn("failure detected: \n", sparkPlan)
```
Comment (birdstorm) [inline core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala]: will automatically output explain result when query fails.

File: core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala [hunk 3]
```diff
@@ -401,9 +418,10 @@ class BaseTiSparkTest extends QueryTest with SharedSQLContext {
         if (!compSqlResult(qSpark, r1, r4, checkLimit)) {
           fail(
             s"""Failed with
-               |TiSpark:\t\t${listToString(r1)}
+               |TiFlash:\t\t${listToString(r4)}
```
Comment (birdstorm) [inline core/src/test/scala/org/apache/spark/sql/BaseTiSparkTest.scala]: Previous output result was not correct @zhexuany

File: core/src/main/scala/com/pingcap/tispark/TiDBRelation.scala [hunk 4]
```diff
@@ -83,28 +82,10 @@ case class TiDBRelation(session: TiSession,
...
-    val sortAgg = TiUtil
-      .planAggregateWithoutPartial(
-        Seq(handlePlan.attributeRef.head), // group by region id
-        Seq(aggExpr),
-        Seq(handlePlan.output.head, aggExpr.resultAttribute), // output <region, handleList>
-        handlePlan
-      )
-      .children
-      .head
+    // TODO: we may optimize by partitioning the result by region.
```
Comment (zhexuany) [inline core/src/main/scala/com/pingcap/tispark/TiDBRelation.scala]: we need create an issue to track this. It is also better to have the issue link.

**Pull request merged**